package.json

```diff
@@ -9,14 +9,16 @@
   },
   "version": "1.0.0",
   "scripts": {
+    "postinstall": "npm run compile:spectypes",
```

Comment:
I would prefer to have the compile step before the test and start commands (e.g.: `"start": "compile:spectypes && ts-node index.ts"`, so that when running the benchmarks locally I'm always testing against the latest compiled spectypes.